apps/docs-app/app/components/f/form/search.gts

```diff
@@ -0,0 +1,334 @@
+    );
+    return things;
+  }
+
+  @action
+  update(key: string, value: unknown) {
+    set(this, key, value);
+  }
+
+  get stringOptionsSource() {
+    return JSON.stringify(this.stringOptions, null, 2);
+  }
+
+  get objectOptionsSource() {
+    return JSON.stringify(this.objectOptions, null, 2);
+  }
+
+  <template>
+    <FreestyleSection @name="Search" as |Section|>
+      <Section.subsection @name="Basic">
+        <FreestyleUsage>
+          <:example>
+            <Search
+              class={{this.class}}
+              @basic={{this.basic}}
+              @binding={{bind this.model "property"}}
+              @clearable={{this.clearable}}
+              @disabled={{this.disabled}}
+              @hideSearchIcon={{this.hideSearchIcon}}
+              @loading={{this.loading}}
+              @minCharacters={{this.minCharacters}}
+              @noResultsLabel={{this.noResultsLabel}}
+              @placeholder={{this.placeholder}}
+              @onChange={{fn log "The value changed to"}}
+              @query={{this.stringQuery}}
+              @readonly={{this.readonly}}
+              @scrollable={{this.scrollable}}
+              @searchTimeout={{this.searchTimeout}}
+            />
+          </:example>
+          <:api as |Args|>
+            <Args.String
+              @name="class"
+              @description="The class to apply to the input. Note that this is not an argument but rather a class applied directly to the input"
+              @value={{this.class}}
+              @onInput={{fn this.update "class"}}
+              @options={{this.classOptions}}
+            />
+            <Args.Bool
+              @name="basic"
+              @defaultValue={{false}}
+              @description="When true, the border will be removed"
+              @value={{this.basic}}
+              @onInput={{fn this.update "basic"}}
+            />
+            <Args.String
+              @name="binding"
+              @description="Create a two-way binding with the value"
+              @value={{this.model.property}}
+              @onInput={{fn this.update "model.property"}}
+            />
+            <Args.Bool
+              @name="clearable"
+              @defaultValue={{false}}
+              @description="When true, adds a clear button"
+              @value={{this.clearable}}
+              @onInput={{fn this.update "clearable"}}
+            />
+            <Args.Bool
+              @name="disabled"
+              @defaultValue={{false}}
+              @description="When true, the input will be disabled"
+              @value={{this.disabled}}
+              @onInput={{fn this.update "disabled"}}
+            />
+            <Args.Bool
+              @name="hideSearchIcon"
+              @defaultValue={{false}}
+              @description="When true, the search icon will be hidden"
+              @value={{this.hideSearchIcon}}
+              @onInput={{fn this.update "hideSearchIcon"}}
+            />
```

Comment:
Note - this also hides the loading indicator. Is there any way around that, or is that something that should be documented?